lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1074,7 +1080,7 @@ def mouse_init(self, rotate_btn=1, zoom_btn=3):
             c3 = canv.mpl_connect('button_release_event', self._button_release)
             self._cids = [c1, c2, c3]
         else:
-            warnings.warn(
+            _log.warning(
```

Comment:
Is this not also a case the user should do something about (and thus `_warn_external`).